test-case-reporting/test/e2e.test.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Knex from 'knex';
+import request from 'supertest';
+
+import {Database, dbConnect} from '../src/db';
+import {app} from '../index';
+import {getFixture} from './fixtures';
+import {setupDb} from '../src/setup_db';
+
+jest.mock('../src/db', () => ({
```

Comment:
nit: I like to name test files after the src files, i.e. `db.test.ts`, because you might want to split your e2e tests across multiple files.

Reply:
What would that look like? Would it be like
```
import {Database, dbConnect} from '../src/db.test';

jest.mock('../src/db.test', () => ({
  dbConnect: () =>
    Knex({
      client: 'sqlite3',
      connection: ':memory:',
      useNullAsDefault: true,
    }),
}));
```

Reply:
Oh, do you mean renaming the file from `e2e.test` to `db.test`?

Reply:
@estherkim the plan is to have unit tests mirror the module names, but many of the apps in this repo use `e2e.test.ts` as the end-to-end test file.

Reply:
Yeah I meant renaming the test file itself. Ok, sg.

Reply:
Understood, resolving conversation.